### PR TITLE
Added The Little Elixir and OTP Guidebook!

### PR DESCRIPTION
**Pull request by benjamintanweihao**

### CODE REVIEW

File: learning.markdown [hunk 1]
```diff
@@ -48,6 +48,15 @@ This book shows you how to write simple Elixir programs by teaching one skill at
...
+-– by Benjamin Tan Wei Hao"><img src="http://www.manning.com/tanweihao/tanweihao_cover150.jpg" alt="The Little Elixir and OTP Guidebook cover" width="190" height="238" /></a>
+
+Elixir is a functional programming language built on the Erlang virtual machine. It combines the productivity and expressivity of Ruby with the concurrency and fault-tolerance of Erlang. Elixir makes full use of Erlang's powerful OTP library, which many developers consider the source of Erlang's greatness, so you can have mature, professional-quality functionality out of the gate. Elixir's support for functional programming makes it a great choice for highly distributed event-driven applications like IoT systems.
+
+The Little Elixir & OTP Guidebook gets you started programming applications with Elixir and OTP. You begin with a quick overview of effectively. Then, you'll dive straight into OTP and learn how it helps you build scalable, fault-tolerant and distributed applications through several fun examples. Come rediscover the joy of programming with Elixir and remember how it feels like to be a beginner again.
```
Comment (josevalim) [inline learning.markdown]: "You begin with a quick overview of effectively" are we missing a word?

File: learning.markdown [hunk 2]
```diff
@@ -48,6 +48,15 @@ This book shows you how to write simple Elixir programs by teaching one skill at
 
 <div class="clear"></div>
 
+<h4 class="resource">The Little Elixir and OTP Guidebook</h4>
+
+<a class="cover" href="http://www.manning.com/tanweihao/" title="The Little Elixir and OTP Guidebook
+-– by Benjamin Tan Wei Hao"><img src="http://www.manning.com/tanweihao/tanweihao_cover150.jpg" alt="The Little Elixir and OTP Guidebook cover" width="190" height="238" /></a>
+
+Elixir is a functional programming language built on the Erlang virtual machine. It combines the productivity and expressivity of Ruby with the concurrency and fault-tolerance of Erlang. Elixir makes full use of Erlang's powerful OTP library, which many developers consider the source of Erlang's greatness, so you can have mature, professional-quality functionality out of the gate. Elixir's support for functional programming makes it a great choice for highly distributed event-driven applications like IoT systems.
```
Comment (josevalim) [inline learning.markdown]: I think we can skip this paragraph, as it is only about the language, and not about the book. What do you think?

Comment (benjamintanweihao) [inline learning.markdown]: Sure thing!

File: learning.markdown (hunk 1 above)
Comment (benjamintanweihao) [inline learning.markdown]: Also, I think it should have been:

You begin with a quick overview of the Elixir language syntax, along with
just enough functional programming to use it effectively.

Shall I send another PR?

Comment (josevalim) [inline learning.markdown]: You can send another PR or edit this one and force push.

**Comment by josevalim** [comment]
Let me know when this is good to go!

**Comment by benjamintanweihao** [comment]
Ok! Will get it by end of the day.

**Comment by benjamintanweihao** [comment]
@josevalim done!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
